aimmo-game-worker/test-initialise.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
```

Comment:
This was being used after development of the game worker, before the game itself starting sending requests, for basic testing